[2022-09-08T18:49:51.675+0500] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: expectation_dag.expect_mean manual__2022-09-08T13:49:44.052113+00:00 [queued]>
[2022-09-08T18:49:51.680+0500] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: expectation_dag.expect_mean manual__2022-09-08T13:49:44.052113+00:00 [queued]>
[2022-09-08T18:49:51.680+0500] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T18:49:51.680+0500] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-08T18:49:51.680+0500] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T18:49:51.694+0500] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): expect_mean> on 2022-09-08 13:49:44.052113+00:00
[2022-09-08T18:49:51.700+0500] {standard_task_runner.py:52} INFO - Started process 1069827 to run task
[2022-09-08T18:49:51.703+0500] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'expectation_dag', 'expect_mean', 'manual__2022-09-08T13:49:44.052113+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/expectation_dag.py', '--cfg-path', '/tmp/tmpy0fz5k2s', '--error-file', '/tmp/tmp_csndmoa']
[2022-09-08T18:49:51.703+0500] {standard_task_runner.py:80} INFO - Job 33: Subtask expect_mean
[2022-09-08T18:49:51.741+0500] {task_command.py:371} INFO - Running <TaskInstance: expectation_dag.expect_mean manual__2022-09-08T13:49:44.052113+00:00 [running]> on host shahroze
[2022-09-08T18:49:51.797+0500] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=expectation_dag
AIRFLOW_CTX_TASK_ID=expect_mean
AIRFLOW_CTX_EXECUTION_DATE=2022-09-08T13:49:44.052113+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-08T13:49:44.052113+00:00
[2022-09-08T18:49:51.801+0500] {logging_mixin.py:115} INFO - {
  "meta": {},
  "exception_info": {
    "raised_exception": false,
    "exception_traceback": null,
    "exception_message": null
  },
  "expectation_config": {
    "meta": {},
    "kwargs": {
      "column": "trip_distance",
      "min_value": 0,
      "max_value": 3,
      "result_format": "BASIC"
    },
    "expectation_type": "expect_column_mean_to_be_between"
  },
  "success": true,
  "result": {
    "observed_value": 2.7589909999999995,
    "element_count": 10000,
    "missing_count": null,
    "missing_percent": null
  }
}
[2022-09-08T18:49:51.801+0500] {python.py:173} INFO - Done. Returned value was: None
[2022-09-08T18:49:51.806+0500] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=expectation_dag, task_id=expect_mean, execution_date=20220908T134944, start_date=20220908T134951, end_date=20220908T134951
[2022-09-08T18:49:51.834+0500] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-08T18:49:51.846+0500] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
